docs/walkthroughs/test-dev-dependency.md

```diff
@@ -0,0 +1,113 @@
+# Working with Test and Development Dependency
+
+FOSSA CLI, by default, does not include the unused dependency in analysis. This is expected for scenarios in which, you have a dependency for development, testing, and other purposes - which ultimately are not used in your runtime. FOSSA CLI can infer dependency's inclusion for runtime or distribution by package manager's manifest file. When we are unable to infer if the dependency is only used in runtime, we choose the conservative approach and include them in our analysis reporting. 
+
+CLI's language support documentation (in [strategy's section](./../references/strategies/README.md)), includes whether an analysis of a particular project will by default unused dependency.
+
+## Example 
+
+For the following `package.json` file:
+
+```json
+{
+    "name": "example-project",
+    "dependencies": {
+        "pngjs": "6.0.0",
+        "once": "1.4.0"
+    },
+    "devDependencies": {
+        "mocha": "*"
+    }
+}
+```
+
+### When running `fossa analyze`:
+
+We will report the dependency tree as follows:
+
+```tree
+.
+ |-npm+once$1.4.0 (direct)
+ | \-npm+wrappy$1 (transitive)
+ \-npm+pngjs$6.0.0 (direct)
+```
+
+Note that, we are not reporting "mocha" dependency or any other transitive dependency of mocha. This is intended behavior, as aforementioned, FOSSA CLI, by default only report dependency which is used in your runtime or distribution.
+
+### How to include all dependencies?
+
+You can use the `--include-unused-deps` flag to explicitly include all _discovered_ dependencies found. This will include all unused dependencies (development, configuration, etc.)
```

Comment:
What does discovered found? I think this would be better if you said something like the following.
```suggestion
You can use the `--include-unused-deps` flag to explicitly include all dependencies, regardless of how they are used. This will include all dependencies (development, configuration, etc.)
```